assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidgetGA4.js

```diff
@@ -103,12 +111,13 @@ function DashboardTopEarningPagesWidgetGA4( { WidgetNull, Widget } ) {
 
 			<Grid className="googlesitekit-padding-top-0">
 				<SettingsNotice
+					Icon={ InfoIcon }
 					notice={ __(
 						'Top earning pages are not yet available in Google Analytics 4.',
```

Comment:
Not part of the change here but this title/heading should not have a full-stop (as in the design).

Reply:
Updated, thank you!